service/src/main/java/bio/terra/workspace/service/resource/referenced/flight/create/CreateReferenceResourceFlight.java

```diff
@@ -21,10 +21,7 @@ public CreateReferenceResourceFlight(FlightMap inputParameters, Object beanBag)
 
     FlightBeanBag appContext = FlightBeanBag.getFromObject(beanBag);
 
-    // Perform access verification
-    addStep(new ValidateReferenceStep(appContext), RetryRules.cloud());
```

Comment:
We also have Stairway idempotency tests for this step (https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/test/java/bio/terra/workspace/service/resource/referenced/ReferencedResourceServiceTest.java#L250 and the `undo` test below that), can you either remove or comment those lines out as well?

Reply:
I don't understand what to comment out. The tests still pass, and are targeting a flight that is still idempotent, right?